src/app/tests/suites/TestEvents.yaml

```diff
@@ -62,6 +58,18 @@ tests:
       response:
           value: { arg1: 1, arg2: 2, arg3: true }
 
+    - label: "Read the event with eventNumber set to the event value"
+      command: "readEvent"
+      event: "TestEvent"
+      eventNumber: eventNumber
+      response:
+          value: { arg1: 1, arg2: 2, arg3: true }
+
```

Comment:
have you triggered any command that generate a new event so that next read event command can get event number + 1

Reply:
The goal of the next test step is to validate that no events are returned when using `eventNumber + 1`, while when using `eventNumber` there should be the previous event.

Reply:
i see, thanks for clarification